release.nix

```diff
@@ -80,6 +58,10 @@ in
       tarball-dhall-nixpkgs
       tarball-dhall-openapi
       tarball-dhall-yaml
+
+      prelude-dhall-docs
+      test-dhall-docs
+#     kubernetes-dhall-docs
```

Comment:
Note that I plan on fixing this before merging.  I commented it out temporarily because it failed due to too many open file handles when building.

Reply:
I had that error, and the solution was to rebuild when there are no executing Jobs on Hydra